src/pages/projects/[name].tsx

```diff
@@ -1,7 +1,547 @@
-import { PageProps } from "gatsby";
```

Comment:
it looks same like src/pages/projects/{project.name}.tsx. If so can we use git mv instead?